test/controllers/employments/AddEmploymentControllerSpec.scala

```diff
@@ -313,7 +313,7 @@ class AddEmploymentControllerSpec
     "redirect user to payroll number page" when {
       "yes is selected" in {
         val sut = createSUT
-        when(addEmploymentJourneyCacheService.cache(mockEq(AddEmployment_RecewivedFirstPayKey), any())(any()))
+        when(addEmploymentJourneyCacheService.cache(mockEq(AddEmployment_ReceivedFirstPayKey), any())(any()))
```

Comment:
not relevant to the PR changes but I assume the any() functions give a value of any permitted type?

Reply:
@PerrettJ4 yep. spot on

Reply:
The reason for any is because most of methods will accept some implicit carrier that holds data.